client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -173,6 +176,12 @@ export const useStocktakeColumns = ({
         description: 'description.snapshot-num-of-packs',
         align: ColumnAlign.Right,
         Cell: PositiveNumberCell,
+        isError: row => {
+          let rows = 'lines' in row ? row.lines : [row];
+          return rows.some(
+            r => getError(r)?.__typename === 'SnapshotCountCurrentCountMismatch'
```

Comment:
This is how the context is used to get an error for a row (recordWithId), in this case if the group by item has a line with an error or if line is erroneous, mark this cell as error